### DISCUSSION TITLE
possible to define variable in mdx?

### DISCUSSION THREAD

**Discussion by nshen** [hashicorp/next-mdx-remote#469] [Q&A]
```
export const name = 'world'

# Hello {name.toUpperCase()}

```
I got

<img width="582" alt="image" src="https://github.com/user-attachments/assets/f080f0e0-c3ed-4561-8358-a0e3b86388f7">

**Comment by talatkuyuk** [comment]
Yes it is possible, just to enable exports via the mdx option `useDynamicImport: true`
